protocol/x/ratelimit/keeper/keeper.go

```diff
@@ -77,6 +77,15 @@ func (k Keeper) ProcessWithdrawal(
 			)
 		}
 
+		metrics.EmitTelemetryWithLabelsForExecMode(
```

Comment:
Why put this in the for loop?

If there's no `CapacityList` for the denom then we won't execute this logic. Should probably place this at the start of this function?

Reply:
Additionally can we also add a counter `withdrawal_rate_limited` [here](https://github.com/dydxprotocol/v4-chain/blob/572244641e56a6446707f913a4f2c0ce7c6f9fa9/protocol/x/ratelimit/keeper/packet.go#L173) so we know how many times withdrawals are getting rate-limited. Thanks!

Reply:
ah... this was an error on my part. ')' looked like '}' . Will fix